README.md

```diff
@@ -88,6 +89,8 @@ cd axolotl
 pip3 install packaging
 pip3 install -e .[flash-attn]
 pip3 install -U git+https://github.com/huggingface/peft.git
+# optionally install deepspeed
```

Comment:
See https://github.com/OpenAccess-AI-Collective/axolotl/issues/598, `deepspeed` doesn't appear to be optional anymore. If it's not installed, Axolotl won't run.